models/ed/R/model2netcdf.ED2.R

```diff
@@ -643,11 +641,35 @@ put_T_values <- function(yr, nc_var, out, lat, lon, begins, ends, ...){
   
   
   # ----- define ncdf dimensions
+  #### setup output time and time bounds
+  ## Create a date vector that contains each day of the model run for each output year (e.g. "2001-07-15", "2001-07-16"....)
+  ## and which is the correct length for each full or partial year
+  output_date_vector <- seq(begins, by = "day", length.out = length(lubridate::yday(begins):ends))
+  ## Calculate model output frequency per day (e.g. 0.02083333)
+  model_timestep_s <- length(output_date_vector)/length(out[[1]])
+  iter_per_day <- round(1/model_timestep_s) ## e.g. 48
+  ## Create a timesteps vector (e.g. 0.00000000 0.02083333 0.04166667 0.06250000 0.08333333 0.10416667 ...)
+  timesteps <- seq(0, 0.99, 1/iter_per_day)
```

Comment:
In case my comment disappeared somewhere since the revision, I would change this to explicitly drop the last value. Otherwise, if we have more than 100 timesteps per day (not common, but also not unreasonable), this will fail.

`head(seq(0, 1, by = 1/iter_per_day), -1)`

Reply:
Yup, I made this change thanks!